.goreleaser.yml

```diff
@@ -182,25 +177,28 @@ dockers:
   dockerfile: images/hybrid/Dockerfile
   extra_files:
     - "images/hybrid/"
+  skip_push: {{ if .Env.SKIP_DOCKER_PUSH == 'true' }}true{{ else }}false{{ end }}
 
 # Build plugin-compiler multiarch
 - ids:
     - std
   image_templates:
-    - "tykio/tyk-plugin-compiler:{{ .Tag }}"
+    - "tykio/tyk-plugin-compiler{{ if .Env.SKIP_DOCKER_PUSH == 'true' }}-el7{{ end }}:{{ .Tag }}"
```

Comment:
I think this if is the only one required, the remaining could just use the env var directly?

Reply:
env var would be `1.15` or `1.15-el7`. that would change the plugin compiler name to `tykio/tyk-plugin-compiler-1.15` and `tykio/tyk-plugin-compiler-1.15-el7`?

Reply:
Missed this earlier. It will have to be `"tykio/tyk-plugin-compiler:{{ .Tag }}{{ if .Env.SKIP_DOCKER_PUSH == 'true' }}-el7{{ end }}"`
We do not have a different repo for the el7 plugin compiler. We just need the tag to indicate it.

Reply:
Also, that variable name is looking weird. Perhaps it could be called `EL7_BUILD` instead of `SKIP_DOCKER_PUSH`? I can make this change.